conventions.md

```diff
@@ -85,7 +86,7 @@ The representations of speech sounds in PHOIBLE do not make use of the tie bar (
 # Ordering of diacritics and modifier letters
 *This section makes a distinction between “combining characters” and “spacing modifier letters”, which most linguists would group together under one term “diacritics”. Briefly, combining characters are diacritics that sit above or below the glyph they modify, or are superimposed on it (e.g., the tilde above a nasalized vowel ~~`̃`~~ is a combining character, encoded as ==0303== in the Unicode standard). Spacing modifier letters are diacritics that sit next to the glyph they modify (e.g., the superscript `ʰ` used to indicate aspiration is a modifier letter, encoded as ==02B0==).*
 
-Each segment type that is composed of more than one character is first normalized into a canonical decomposition form that adheres to the [Unicode Normalization Form D (NFD)](http://unicode.org/reports/tr15/). However, the NFD algorithm does not define a prescribed order for all possible pairs of combining characters. For example, ==0301== (combining acute accent) and ==0308== (combining diaresis) are not re-ordered with respect to one another by the NFD algorithm, regardless of which order they occurred in the input string. PHOIBLE imposes an ordering that is consistent with NFD, but sets an explicit order for all combining characters used in the database. A rigid ordering for spacing modifier letters is also enforced. The chosen ordering is guided by the linguistic literature and typographical concerns, since to the best of our knowledge the IPA does not provide a recommended ordering for diacritics.
+Each representation of a phoneme is first normalized into a canonical decomposition form that adheres to the [Unicode Normalization Form D (NFD)](http://unicode.org/reports/tr15/). However, the NFD algorithm does not define a prescribed order for all possible pairs of combining characters. For example, ==0301== (combining acute accent) and ==0308== (combining diaresis) are not re-ordered with respect to one another by the NFD algorithm, regardless of which order they occurred in the input string. PHOIBLE imposes an ordering that is consistent with NFD, but sets an explicit order for all combining characters used in the database. A rigid ordering for spacing modifier letters is also enforced. The chosen ordering is guided by the linguistic literature and typographical concerns, since to the best of our knowledge the IPA does not provide a recommended ordering for diacritics.
```

Comment:
Change is from "Each segment type that is composed of more than one character..." to "Each representation of a phoneme..."  I did this for 2 reasons: 

1. we don't define "segment type" anywhere, and I think it's sort of a legacy term that we don't need to maintain, and
2. we actually denormalize everything, not just strings comprising more than one character.  In fact, that is the whole reason that c-cedilla is a pain in our butts.